fabfile.py

```diff
@@ -36,17 +36,17 @@ def build_static():
     local("./node_modules/.bin/gulp")
 
 
-def clear_memcached():
+def clear_redis_cache():
     try:
-        cache.init(config.MEMCACHED_SERVERS)
+        cache.init(config.REDIS_HOST, config.REDIS_PORT, config.REDIS_NAMESPACE)
         cache.flush_all()
```

Comment:
No bug in the lib. Just a silly error on my part. Cache should be initialized as:

`cache.init(host='127.0.0.1', port=6379, namespace='AB')`
or
`cache.init('127.0.0.1', 6379, 0, 'AB')`

to avoid setting the 'AB' string for `db_number` argument in the init function. I should change this for other cache inits as well.